### PR TITLE
refactor(config): use templates (batch 1)

### PR DESCRIPTION
**Pull request by blhoward2**
Adds an initial master template and templates for Homeseer and HomePro devices.

### CODE REVIEW

**Comment by blhoward2** [comment]
@AlCalzone Lint doesn't like the master template. It seems to be ignoring the ../ Thoughts?

**Comment by blhoward2** [comment]
@AlCalzone Fixed and ready to go

File: packages/config/config/devices/0x0001/zdw103.json [hunk 1]
```diff
@@ -17,318 +17,72 @@
...
-			"label": "Group 2",
-			"description": "Tap twice",
+			"label": "Double Tap",
 			"maxNodes": 5
 		},
 		"3": {
-			"label": "Group 3",
-			"description": "Tap three time",
+			"label": "Triple Tap",
+
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0001/zdw103.json]:
```suggestion
```

File: packages/config/config/devices/0x0001/zrw230.json [hunk 2]
```diff
@@ -18,176 +18,44 @@
...
-			"label": "Tapping paddle once",
-			"maxNodes": 5,
-			"isLifeline": true
+			"label": "Single Tap",
+			"maxNodes": 5
 		},
 		"2": {
-			"label": "Tapping paddle twice",
-			"maxNodes": 5,
-			"isLifeline": true
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0001/zrw230.json:28]: Do these commands reach the controller without the groups marked as lifelines?

File: packages/config/config/devices/0x0001/ztm230.json [hunk 3]
```diff
@@ -17,244 +17,86 @@
 	},
 	"associations": {
 		"1": {
-			"label": "Group 1",
-			"description": "Group 1 left paddle",
-			"maxNodes": 5,
-			"isLifeline": true
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0001/ztm230.json:23]: dito

File: packages/config/config/devices/0x000c/hs-ms100.json [hunk 4]
```diff
@@ -1,24 +1,25 @@
...
 	"firmwareVersion": {
 		"min": "0.0",
 		"max": "255.255"
 	},
-	"supportsZWavePlus": true,
 	"paramInformation": {
 		"12": {
-			"label": "Motion sensor sensitivity",
+			"label": "Motion Sensor Sensitivity",
+			"description": "Higer values increases sensitivity",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x000c/hs-ms100.json]:
```suggestion
			"description": "Higher values increase sensitivity",
```

File: packages/config/config/devices/0x000c/hs-wv100.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+			"zwaveAllianceId": 1845
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Group 1",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x000c/hs-wv100.json]:
```suggestion
			"label": "Lifeline",
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Just a few nits I'm going to change myself. And these two questions:

File: packages/config/config/devices/0x0001/zrw230.json (hunk 2 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0001/zrw230.json:28]: I'm not actually sure. I'll re-add them.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**